src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6588,4 +6588,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_UnreadRecordParameter_Title" xml:space="preserve">
     <value>Parameter is unread. Did you forget to use it to initialize the property with that name?</value>
   </data>
-</root>
+  <data name="IDS_FeatureNameofAccessInstanceMembersInAllContexts" xml:space="preserve">
+    <value>nameof can access instance members in all contexts</value>
```

Comment:
>nameof can access instance members in all contexts [](start = 11, length = 50)

Perhaps "instance member access in `nameof`"? We don't normally phrase these with "can" or similar in them.